addons/website_customer/views/website_customer_templates.xml

```diff
@@ -201,6 +221,9 @@
                     <a class="media-heading" t-attf-href="/customers/#{slug(reference)}">
                         <span t-field="reference.self"/>
                     </a>
+                    <t t-if="partner.industry_id">
+                        <span class="label label-default"><t t-esc="partner.industry_id.name"/></span>
```

Comment:
The Sector of activity of the Partner itself is missing in the details. It could be displayed next to its name as a badge, or in the left sidebar with the address.